subsys/bluetooth/host/classic/at.h

```diff
@@ -91,7 +91,7 @@ typedef int (*handle_cmd_input_t)(struct at_client *at, struct net_buf *buf,
 struct at_client {
 	char *buf;
 	uint8_t pos;
-	uint8_t buf_max_len;
```

Comment:
what case would cause that issue？

what is the largest length of AT cmd in spec？